docs/tutorials/aws.md

```diff
@@ -84,7 +84,7 @@ spec:
         args:
         - --source=service
         - --source=ingress
-        - --domain-filter=external-dns-test.my-org.com. # will make ExternalDNS see only the hosted zones matching provided domain, omit to process all available hosted zones
+        - --domain-filter=external-dns-test.my-org.com # will make ExternalDNS see only the hosted zones matching provided domain, omit to process all available hosted zones
```

Comment:
Out of curiosity, is there a branch that is setup for those types of changes that PRs could be created against to allow PR authors the ability to update the docs with a stable 'release' process and limit your need to update them?